rotkehlchen/chain/ethereum/modules/eth2.py

```diff
@@ -375,9 +382,14 @@ def add_validator(self, validator_index: Optional[int], public_key: Optional[Eth
                     msg = f'Missing key entry for {msg}.'
 
                 raise RemoteError(f'Failed to query beaconcha.in for validator data due to: {msg}') from e  # noqa: E501
-
         # by now we have a valid index and pubkey. Add to DB
-        dbeth2.add_validators([Eth2Validator(index=valid_index, public_key=valid_pubkey)])
+        dbeth2.add_validators([
+            Eth2Validator(
+                index=valid_index,
+                public_key=valid_pubkey,
+                ownership_proportion=ownership_proportion,
```

Comment:
Hmmm did you remove the proportion changes in the get_balances() function? I can't find them. The get_balances() of eth2 should show us the balances in ETH2 per validator along with the proper percentage, right?